sdk/core/azure-core-management/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-08-07)
```

Comment:
Would this also be released as part of core release pipeline?

Reply:
Yup!